src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -68,7 +68,8 @@ private[avro] class AvroRelation(
    */
   override def dataSchema: StructType = maybeDataSchema match {
     case Some(structType) => structType
-    case None => SchemaConverters.toSqlType(avroSchema).dataType.asInstanceOf[StructType]
+    case None => SchemaConverters.toSqlType(avroSchema,
+        withAliasFields).dataType.asInstanceOf[StructType]
```

Comment:
Style nit: don't break in the middle of a function call.  wrap the whole line instead.